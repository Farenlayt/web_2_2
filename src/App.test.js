import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Application from './components/Application/Application';
import FavourCitiesHeader from "./components/FavourCitiesHeader/FavourCitiesHeader";
import {Provider} from "react-redux";
import MajorCityWrapper from "./components/MajorCityWrapper/MajorCityWrapper";
import WeatherBody from "./components/WeatherBody/WeatherBody";
import FavourCitiesWrapper from "./components/FavourCitiesWrapper/FavourCitiesWrapper";
import Error from "./components/Error/Error";
import Loader from "./components/Loader/Loader";
import storage from "./store/FakeStorage.js"


it('Renders is ok.', function() {
    const div = document.createElement('div');
    ReactDOM.render(<Application/>, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('Empty major is ok.', function() {
    const tree = renderer.create(
        <Provider store={storage}>
            <Application/>
        </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });