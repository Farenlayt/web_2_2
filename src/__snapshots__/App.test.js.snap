// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty major is ok. 1`] = `
<div
  className="Application"
>
  <div>
    <div
      className="major-header"
    >
      <label>
        Погода здесь
      </label>
      <form
        onSubmit={[Function]}
      >
        <input
          className="btn"
          type="submit"
          value="Обновить геолокацию"
        />
      </form>
    </div>
    <div
      className="major-loader"
    >
      Подождите, данные загружаются
    </div>
  </div>
  <div>
    <div
      className="favour-header"
    >
      <form
        onSubmit={[Function]}
      >
        <div
          className="favour-header-text"
        >
          Избранное
        </div>
        <div>
          <input
            className="input-field"
            id="search-input"
            placeholder="Добавить новый город"
            type="text"
          />
          <input
            className="btn"
            type="submit"
            value="Добавить новый город"
          />
        </div>
      </form>
    </div>
    <div
      className="favour-city-wrapper"
    />
  </div>
</div>
`;
